system/global.nix

```diff
@@ -65,6 +65,7 @@
 
   boot.binfmt.emulatedSystems = [
     "aarch64-linux"
+    "armv7l-linux"
   ];
```

Comment:
I think this whole block should be moved to be a node-specific setting for the build servers, as we really don't do much with emulation support on the compute machines.